ooiui/templates/science/plotting.html

```diff
@@ -535,11 +535,10 @@
       parameterhtml += "</optgroup>"
 
       $("div#yvar0-selection-default > div.form-group > select").html(parameterhtml);
-      if (refreshYVar1){
-        $("div#yvar1-selection > div.form-group > select").html(parameterhtml);
-      }
```

Comment:
@birdage @DanielJMaher this was in here for Interpolated plotting so I think removing it breaks that plot